.ebextensions/db-migrate.config

```diff
@@ -0,0 +1,19 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/post/10_post_migrate.sh":
```

Comment:
what's the `10_` refer to? Just trying to track what's going on

Reply:
AWS uses this dir to run other scripts. The scripts run in sort order post deployment. We prefixed the file with ```10_``` so it doesn't interfere with those other scripts.